resources/assets/less/bem/osu-md.less

```diff
@@ -262,6 +263,7 @@
     margin-bottom: @md-paragraph-space;
     line-height: 1.5;
 
+    .@{_top}--group &,
```

Comment:
change `@md-paragraph-space` to use variable instead (`--paragraph-space: @md-paragraph-space`) and set the override at block level modifier style